src/Application.php

```diff
@@ -174,6 +174,37 @@ public function add(Command $command, string $alias = '', bool $default = false)
         return $this;
     }
 
+    /**
+     * Set the default command.
+     *
+     * @param string $commandName The name of the default command
+     *
+     * @return self The application
+     *
+     * @throws InvalidArgumentException If the specified command name does not exist
+     */
+    public function defaultCommand(string $commandName): self
+    {
+        if (!isset($this->commands[$commandName])) {
+            throw new InvalidArgumentException(sprintf('Command "%s" does not exist', $commandName));
+        }
+
+        $this->default = $commandName;
+
+        return $this;
+    }
+
+    /**
+     * Get the default command.
+     *
+     * @return string|null The name of the default command, or null if not set
+     */
+    public function getDefaultCommand(): ?string
```

Comment:
there might be always a default command i think as it is set in constructor

Reply:
I just use it as a **getter** to get the default command cauz i need it in test

Reply:
no i mean about the nullable string in return type. can be just string i guess. :)

Reply:
Yeah maybe but if command is not default it's should return null right?

Reply:
isn't default set in construct already? `__default__`